src/app/forms/templates/search/single-search-result.template.ts

```diff
@@ -14,42 +15,42 @@ export function createSingleSearchResult(systemNumber?: string): FormNode {
       {
         name: 'vin',
         label: 'Vehicle identification number (VIN)',
-        value: '',
         type: FormNodeTypes.CONTROL,
         viewType: FormNodeViewTypes.STRING
       },
       {
         name: 'vrm',
         label: 'Vehicle registration mark (VRM)',
-        value: '',
         type: FormNodeTypes.CONTROL,
         viewType: FormNodeViewTypes.STRING
       },
+      {
+        name: 'trailerId',
+        label: 'Trailer ID',
+        type: FormNodeTypes.CONTROL,
+        validators: [{ name: ValidatorNames.HideIfEmpty, args: 'trailerId' }]
```

Comment:
That is already the case, see line 31.

`trailerId` is only populated on TRL records. Thus, when that's `undefined`, this field will not render.

If you pull the changes, you can check it out for yourself.

Reply:
My point was that at the moment you don't render trailerId if it isn't on the tech record even if the vehicle type is a trailer. I feel like it should render the trailerId attribute even if it isn't populated in the record (**for a vehicle type of trailer**) and just have a dash in like the rest of the fields.

With trailerId:
![image](https://user-images.githubusercontent.com/92087051/189162532-882278ce-68a7-422a-9d6f-ea671095912b.png)

Without trailerId:
![image](https://user-images.githubusercontent.com/92087051/189162184-1a35a547-1432-4494-ae44-80fdcff0879e.png)

Reply:
The ticket is a bit ambiguous on this detail. I agree with Aaron that we keep the row visible, but with a dash. It will then be  consistent with the other fields VRM, Make, and Model.

Reply:
I understand and agree. Suggestion applied.